runtime/doc/vim9.txt

```diff
@@ -1771,16 +1770,17 @@ an empty container, do not use null_<type> in a comparison: >
 	F(null_list)	# output: "null"
 	F([])		# output: "not null, empty"
 	F([''])		# output: "not null, not empty"
+<
 The above function takes a `list of strings` and reports on it.
 Change the above function signature to accept different types of arguments: >
 	def F(arg: list<any> = null_list)   # any type of list
 	def F(arg: any = null)		    # any type
 <
-In the above example, where the goal is to distinguish a null list from an
-empty list, comparing against `null` instead of `null_list` is the correct
-choice. The basic reason is because "null_list == null" and "[] != null".
-Comparing to `null_list` fails since "[] == null_list". In the following section
-there are details about comparison results.
+NOTE: In the above example, the arg was assigned to a null container,
+comparing against `null` instead of `null_list` is ok. The basic reason is
```

Comment:
This proposed statement is misleading. Using `arg == null`, and not `arg == null_list`, is required for the example; changing the compare changes the output. **This change breaks the example**.

The whole idea of the example is to show the programmer that there are issues with "null container" versus "empty container"; why do you take out the most important part of the description? That is what this is all about. If the programmer wants them to be different, **which is the way most (all?) strongly typed programming languages work**, the help text should describe how to do it.

Reply:
no, it is just to clear the leading and why that example came true.